### PR TITLE
[KEYCLOAK-7552] - New spring security quickstart and changes to app-authz-springboot

### PR DESCRIPTION
**Pull request by pedroigor**
Depends on https://github.com/keycloak/keycloak/pull/5391

### CODE REVIEW

**Comment by abstractj** [comment]
@matthewhelmke if you find some time this week. Could you please review the README file?

File: app-authz-spring-security/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+* **/protected**, where access to this page is based on the evaluation of permissions associated with a resource **Protected Resource** in <span>Keycloak</span>. Basically,
+any user with a role *user* is allowed to access this page.
+
+* **/protected/premium**, where access to this page is based on the evaluation of permissions associated with a resource **Premium Resource** in <span>Keycloak</span>. Basically,
+only users with a role *user-premium* is allowed to access this page.
+
+* **/protected/alice**, where access to this page is based on the evaluation of permissions associated with a resource **Alice Resource** in <span>Keycloak</span>. Basically,
+only user *alice* is allowed to access this page.
+
+The home page (home.ftl) also demonstrates how to use a ``AuthorizationContext`` instance to check for user`s permissions and hide/show
```
Comment (matthewhelmke) [inline app-authz-spring-security/README.md]: There is a back tick in "user`s" that should instead be an apostrophe, "user's".

File: app-authz-spring-security/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+All you need to build this project is Java 8.0 (Java SDK 1.8) or later and Maven 3.1.1 or later.
+
+
+Configuration in <span>Keycloak</span>
+-----------------------
+
+Prior to running the quickstart you need to create a `realm` in <span>Keycloak</span> with all the necessary configuration to deploy and run the quickstart.
+
+The following steps show how to create the realm required for this quickstart:
```
Comment (matthewhelmke) [inline app-authz-spring-security/README.md]: I suggest this as a standard introduction to a procedure:
To create the realm required for this quickstart, follow these steps:

Comment (matthewhelmke) [reply app-authz-spring-security/README.md]: Oh, also, use an ordered list, with numbers, instead of the bullets.

**Comment by pedroigor** [comment]
@matthewhelmke, thanks. Hope it is fine now.

**Comment by matthewhelmke** [comment]
@pedroigor I was impressed. It is clearly written.

**Pull request merged**